Composer/packages/extension/package.json

```diff
@@ -20,7 +20,6 @@
     "@types/path-to-regexp": "^1.7.0",
     "@types/tar": "^4.0.3",
     "json-schema": "^0.2.5",
-    "rimraf": "^3.0.2",
```

Comment:
shouldn't it remain a dev dependency since it's used in the build scripts? counting on it being in the workspace's hoisted deps seems wonky